### ISSUE TITLE
Crash report: `Error: No V2 factory address on this chain`

### ISSUE THREAD

**Issue by Elham3030** [Uniswap/interface#3762]
# URL
  
https://app.uniswap.org/#/pool/v2?chain=mainnet

## Error

```
Error: No V2 factory address on this chain
```

## Stacktrace

```
Error: No V2 factory address on this chain
    at L (https://app.uniswap.org/static/js/main.e43a79ef.chunk.js:1:274135)
    at https://app.uniswap.org/static/js/main.e43a79ef.chunk.js:1:663495
    at Array.map (<anonymous>)
    at https://app.uniswap.org/static/js/main.e43a79ef.chunk.js:1:663445
    at Object.xa [as useMemo] (https://app.uniswap.org/static/js/3.f05ab84b.chunk.js:2:2098937)
    at https://app.uniswap.org/static/js/3.f05ab84b.chunk.js:2:2013478
    at Tj (https://app.uniswap.org/static/js/main.e43a79ef.chunk.js:1:663423)
    at oa (https://app.uniswap.org/static/js/3.f05ab84b.chunk.js:2:2094204)
    at za (https://app.uniswap.org/static/js/3.f05ab84b.chunk.js:2:2103771)
    at Yo (https://app.uniswap.org/static/js/3.f05ab84b.chunk.js:2:2147495)
```

## Device data

```json
{
  "ua": "Mozilla/5.0 (Linux; Android 5.1.1; SM-J111F) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.74 Mobile Safari/537.36",
  "browser": {
    "name": "Chrome",
    "version": "95.0.4638.74",
    "major": "95"
  },
  "engine": {
    "name": "Blink",
    "version": "95.0.4638.74"
  },
  "os": {
    "name": "Android",
    "version": "5.1.1"
  },
  "device": {
    "vendor": "Samsung",
    "model": "SM-J111F",
    "type": "mobile"
  },
  "cpu": {}
}
```

**Comment by moodysalem** [comment]
can repro this by just clicking polygon in the network dropdown from the pool page

**Comment by behnammohammadkhani** [comment]
@moodysalem also, we have the same issue for other networks (arbitrum, optimism)

**Comment by behnammohammadkhani** [comment]
@moodysalem 
I fixed the issues by this PR:
[PR #3777](https://github.com/Uniswap/interface/pull/3777)

**Comment by ChenEranranS** [comment]
取消订阅发自我的手机-------- 原始邮件 --------发件人： Vignesh Mohankumar ***@***.***>日期： 2022年6月16日周四 清晨5:08收件人： Uniswap/interface ***@***.***>抄送： Subscribed ***@***.***>主    题： Re: [Uniswap/interface] Crash report: `Error: No V2 factory address on this chain` (Issue #3762)
Closed #3762 as completed.
